k8sversion/k8s_version_test.go

```diff
@@ -27,11 +27,11 @@ func Test_K8sVersion_Latest(t *testing.T) {
 		{
 			name: "case 1",
 			versions: []string{
-				"v1alpha1",
+				"v2alpha1",
 				"v1beta1",
 				"v1alpha2",
 			},
-			latest:       "v1beta1",
+			latest:       "v2alpha1",
```

Comment:
This test will fail yielding v1beta1 as latest. This is incorrect IMO.

Reply:
Can you please provide a new test and not change the existing ones? They are there for a reason, which is to ensure certain cases. When you have a new case, add it.

Reply:
This PR is tho show this behaviour is broken. This test doesn't pass now.